src/api/messages.ts

```diff
@@ -58,6 +58,9 @@ export default class MessagesAPI extends BaseAPI implements m.MessagesClass {
 	send(
 		body: RequestData,
 	): Promise<RequesterResponseInterface<m.MessagesResponse>> {
+		if (!this.client) {
+			throw new Error('No client initialized');
```

Comment:
Not sure if this is the actual behavior you want, but there wasn't much else to do about it. Should probably add some tests to define this behavior.